networkingtoolbox/umbrel-app.yml

```
@@ -0,0 +1,36 @@
+manifestVersion: 1
+id: networkingtoolbox
+name: Networking Toolbox
+tagline: An offline-first toolbox for networking
+category: networking
```

The Developer Tools category fits better since Networking Toolbox is more of a technical utility for developers and sysadmins.

```suggestion
category: developer
```